requirements/classes/php2560/requirements.txt

```diff
@@ -0,0 +1,52 @@
+# add class-specific packages here:
+
+# these are the scipy base packages ( YOU PROBABLY DON'T WANT TO REMOVE THESE )
+dask 
+pandas
+numexpr
+matplotlib
+scipy
+seaborn
+scikit-learn
+scikit-image
+sympy
+cython
+patsy
+statsmodels
+cloudpickle
+dill
+numba
+bokeh
+sqlalchemy
+hdf5
+vincent
+beautifulsoup4
+protobuf
+xlrd
+bottleneck
+pytables
+
+# these are the base r packages in the r-notebook image ( YOU PROBABLY WANT THESE IF YOUR TARGET IN DOCKER IS r_lang OR r_julia )
+r-base=4.0.0
+r-caret=6.*
+r-crayon=1.3*
+r-devtools=2.3*
+r-forecast=8.12*
+r-hexbin=1.28*
+r-htmltools=0.4*
+r-htmlwidgets=1.5*
+r-irkernel=1.1*
+r-nycflights13=1.0*
+r-randomforest=4.6*
+r-rcurl=1.98*
+r-rmarkdown=2.2*
+r-rodbc=1.3*
+r-rsqlite=2.2*
+r-shiny=1.4*
+r-tidyverse=1.3*
```

Comment:
this is the conda one. The CRAN packages was something added when we did the hubs for Ashok.... not sure if you want to maintain a mixture...

Reply:
Oh, lord... conda. Okay, yeah, I'll rework it. I might also get a newer version of R, too. We are a few patch release behind now, and some of the packages throw warnings because of that.